packages/react-router/src/components/Link/Link.tsx

```diff
@@ -8,16 +8,20 @@ export interface Props extends ReactRouterLinkProps {
   url: string;
 }
 
-export default function Link({url, external, children, ...rest}: Props) {
-  let target: string | undefined;
-  let rel: string | undefined;
-  if (external) {
-    target = '_blank';
-    rel = 'noopener noreferrer';
+const IS_EXTERNAL_LINK_REGEX = /^(?:[a-z][a-z\d+.-]*:|\/\/)/;
```

Comment:
Can we make the regex simpler by just looking for `/^https?:\/\/`? I mean do we really need to support any other protocols?

Reply:
@atesgoral I'd expect so yeah, we're looking at using some other schemes that apps can register handlers for like `slack://` which we wouldn't want react-router to try and intercept